test/objects/credential/credential.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2024
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package credential
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/Mirantis/hmc/api/v1alpha1"
+)
+
+const (
+	DefaultName      = "credential"
+	DefaultNamespace = "default"
+)
+
+type Opt func(managedCluster *v1alpha1.Credential)
```

Comment:
can we drop `managedCluster` argument name? it is both redundant and misleading 

and it seems `WithDescription` function is not used, and `DefaultNamespace = "default"` is equal to the `DefaultNamespace = metav1.NamespaceDefault`

Reply:
Such a lame copy-paste error)